### DISCUSSION TITLE
Godd job,Valve

### DISCUSSION THREAD

**Discussion by Leyzik** [ValveSoftware/Dota2-Gameplay#7046] [General]
Good afternoon, dear valve.Whenever I launched a rating, and in the enemy team I came across players on whose account the player is playing the game levels higher on this rating (boosters). And I've always crossed my fingers in the hope that valve will ban such players to hell)). But today I TURNED OUT TO BE A CHEATER, ARE YOU CRAZY? No one has access to my account and no one has ever played on my account with third-party software, I only had access to my account from other computers, because I played Dota 2 in the cyber arena (WITHOUT USING THIRD-PARTY software). I think blocking my native account is wrong. I hope you will understand me and reconsider your decision)

**Comment by AvanaWindows00** [comment]
Looks like a small typo there—probably meant “Good job, Valve”! It’s always great to see Valve getting recognition for their innovations and contributions to gaming. Keep up the impressive work! [royal ascot betting](https://bettinginsight.net/sports-betting-guide/royal-ascot-betting-today/)
